hack/check-ds.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+NAMESPACE="${1:-default}"
+DAEMONSET="${2:-resource-topology-exporter-ds}"
+
+function wait-for-daemonset(){
+    retries=24
+    while [[ $retries -ge 0 ]];do
+        sleep 5
+        ready=$(kubectl -n $NAMESPACE get daemonset $DAEMONSET -o jsonpath="{.status.numberReady}")
+        required=$(kubectl -n $NAMESPACE get daemonset $DAEMONSET -o jsonpath="{.status.desiredNumberScheduled}")
+        if [[ $ready -eq $required ]];then
+            break
+        fi
+        ((retries--))
+        # debug
+        kubectl -n $NAMESPACE describe daemonset $DAEMONSET
```

Comment:
Maybe it's better to print it only in case that the ds isn't ready? instead of each time for every iteration

Reply:
thanks, reworked